app/views/shared/views/_show_topology.html.haml

```diff
@@ -0,0 +1,77 @@
+- tooltipOptions = {"tooltip-placement" => "bottom", "tooltip" => "{{legendTooltip}}"}
```

Comment:
> imo this file is too specific to be in "shared". because there are 3 different topologies, each with its own legend so the below content is too specific for containers and irrelevant to other topologies.

Right. Then I think ill drop the shared view, there isnt enough in common for these topology views to merit a shared view